### PR TITLE
test with 3.9

### PR DESCRIPTION
**Pull request by graingert**
### Description

<describe bug fix or new feature>

Fixes: #

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,16 +4,14 @@ sudo: required
...
 install:
     - "make setup"
     - "pip install ."
 script:
     - "make test"
 matrix:
-    allow_failures:
-        - python: "3.7-dev"
     include:
         - python: "3.6"
```
Comment (graingert) [inline .travis.yml:16]: do we want linting to run on 3.10?

File: .travis.yml [hunk 2]
```diff
@@ -4,16 +4,14 @@ sudo: required
 env:
 python:
     - "3.7"
```
Comment (graingert) [inline .travis.yml:6]: what about 3.6 and 3.10-dev?

**Comment by jreese** [comment]
Hmm, we should actually just go ahead and remove the Travis config, and update the GHA config to include the 3.9 runtime.  Since pylint still doesn't support 3.9, I've been using a [config like this](https://github.com/omnilib/aiosqlite/blob/main/.github/workflows/build.yml) to skip the lint phase on 3.9.  As for 3.10, I believe that GHA has support for that as `"3.10-dev"`, but I haven't tried that anywhere, and would require a more complicated conditional to skip pylint there as well.

**Comment by graingert** [comment]
@jreese pylint 2.8 released recently

**Pull request merged**